beacon-chain/state/state-native/state_trie.go

```diff
@@ -145,7 +151,7 @@ const (
 	denebSharedFieldRefCount     = 7
 	electraSharedFieldRefCount   = 10
 	fuluSharedFieldRefCount      = 11
-	gloasSharedFieldRefCount     = 12
+	gloasSharedFieldRefCount     = 12 // Adds LatestExecutionPayloadBid to the shared-ref set.
```

Comment:
If you remove the payload bid then this comment needs to change?

Reply:
Sorry, i meant to say `Adds pending builder withdrawals to the shared-ref set`, not `LatestExecutionPayloadBid`